src/pages/accounts.tsx

```diff
@@ -67,6 +68,7 @@ accounts.use(loginRequired);
 
 accounts.get("/", async (c) => {
   const owners = await db.query.accountOwners.findMany({
+    where: ne(accountOwners.handle, HOLLO_RELAY_ACTOR_USERNAME),
```

Comment:
Sometimes you've compare `account_owners.id` with `HOLLO_RELAY_ACTOR_ID`, and sometimes `account_owners.handle` with `HOLLO_RELAY_ACTOR_USERNAME`. Is there any difference between both ways?

Reply:
I think this check should be using the ID here, not the username.

Reply:
This may also affect the setup page, which checks the number of actors on the server.

Reply:
Oh, it may affect the NodeInfo stats as well:

https://github.com/fedify-dev/hollo/blob/65c2b117dfe3d2e9d35ca813662fc37b6afb44e8/src/federation/nodeinfo.ts#L18-L28